app/components/post_textbox/post_textbox_base.js

```diff
@@ -106,11 +113,27 @@ export default class PostTextBoxBase extends PureComponent {
             Keyboard.addListener('keyboardDidHide', this.handleAndroidKeyboard);
             BackHandler.addEventListener('hardwareBackPress', this.handleAndroidBack);
         }
+
+        if (this.props.isTimezoneEnabled) {
```

Comment:
I think you should create a function to call this action as the code here and the one in componentWillReceiveProps is practically the same.  With that you also get rid of the eslint rules problem and the need to set async to the lifecycle method.

Reply:
Thanks for the suggestion! I put all that code into a function.